src/Controller/Badge/VersionController.php

```diff
@@ -32,17 +33,21 @@ public function version(
         CreateVersionBadge $createVersionBadge,
         string $repository,
         string $latest,
-        string $format = 'svg'
+        string $format = 'svg',
+        string $style = 'flat',
     ): Response {
-        if (\in_array($request->query->get('format'), $poser->validStyles(), true)) {
-            $format = $request->query->get('format');
-        }
+//        if (!\in_array($request->query->get('style'), $poser->validStyles(), true)) {
+//            $style = Badge::DEFAULT_STYLE;
+//        }
+
+        var_dump($style);
+        die;
```

Comment:
![die](https://media.giphy.com/media/kHcaX3uLU3IME/giphy.gif)

Reply:
This is for testing. I am not getting the actual style parameter I put in the URL. It does not work yet! Can you help?

Reply:
after making the change, restart docker-compose, sometimes the changes are not applied in the container :S

Reply:
as soon as I have a second I throw an eye too ;)

Reply:
I fixed your problem! Can i commit on this branch?

Reply:
Sure

Reply:
I add my commit here: https://github.com/PUGX/badge-poser/pull/733

Reply:
Why exactly did you open a second PR? You should be able to commit to my fork.